docs/miscellaneous/cli.md

```diff
@@ -170,35 +172,39 @@ Flags:
   -h, --help                        help for configure
 ```
 
-### cluster export
```

Comment:
We've used `dev/generate_cli_md.sh` to generate this file, and it seems that the order of these two commands is different there. So if you think this order is better, you can update the order in the generator (and might as well rerun it to confirm that there is no diff)

Reply:
The current order that we get in `cortex cluster` is `up, info, configure, down, export`. The generator apparently wasn't in the correct order. Modified the order and re-ran to make sure there is no diff.